actions/geoip.py

```diff
@@ -95,6 +100,13 @@ def run(self, ip_addresses):
                                       'value': response.isp}
                     details['org'] = {'name': "Org",
                                       'value': response.organization}
+                elif reader_asn:
```

Comment:
Does this need to be `elif` or just `if` is it makes sense to include both type of information?